src/style.css

```
@@ -1 +1,69 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
```

```suggestion
* {
  box-sizing: border-box;
}

html {
    font-family: Avenir, Arial, Helvetica, sans-serif;
    font-weight: 300;
    font-style: normal;
    color: rgba(61, 78, 97, 1);
}
```
* selector use in general only for box-sizing.